shell/browser/serial/serial_chooser_context.cc

```diff
@@ -174,11 +196,19 @@ bool SerialChooserContext::CanStorePersistentEntry(
   if (!port.display_name || port.display_name->empty())
     return false;
 
+  const bool has_bluetooth = port.bluetooth_service_class_id &&
+                             port.bluetooth_service_class_id->IsValid() &&
+                             !port.path.LossyDisplayName().empty();
```

Comment:
Looks like `LossyDisplayName()` is a one-liner that converts `path_` to a std::u16string. We could probably avoid the new string + encoding conversion?

```suggestion
                             !port.path.empty();
```

Reply:
this was done to match chrome but i'm totally impartial so happy to change!

Reply:
Yeah, agreed about the value of matching upstream.

Doing a charconv + instantiating a temporary string just to see if it's empty is so imminently avoidable though, TBH I still vote for for using `port.path.empty()` even though matching Chrome is an important guideline

Reply:
@ckerr done in b2058b51e8!